db/schema.rb

```diff
@@ -99,6 +99,18 @@
     t.index ["state_id"], name: "index_registration_agencies_on_state_id"
   end
 
+  create_table "related_instructions", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "title"
+    t.int4range "hours"
+    t.boolean "elective"
+    t.integer "sort_order"
+    t.uuid "occupation_standard_id", null: false
+    t.integer "default_course_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["occupation_standard_id"], name: "index_related_instructions_on_occupation_standard_id"
+  end
```

Comment:
Could you remove the related_instructions piece out of the schema (including foreign key statement at the end).